homeassistant/components/rainforest_eagle/sensor.py

```diff
@@ -135,6 +135,8 @@ def unique_id(self) -> str | None:
     @property
     def native_value(self) -> StateType:
         """Return native value of the sensor."""
+        if not self.coordinator.is_connected:
+            return None
```

Comment:
Why don't we set `available` to False here?

Reply:
The hub itself is still available. It is just not getting data from the electric meter so the data is has is stale. 

I guess it really depends on what you consider what device this entity to represent. Currently I am representing it from the point of the eagle-200 hub.

Reply:
But I guess the coordinator is limited to data from a single meter. I'll rewrite.